hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -1994,7 +1994,7 @@ possible configurations would overwhelm and obscure the important.
   </property>
   <property>
     <name>hbase.namedqueue.provider.classes</name>
-    <value>org.apache.hadoop.hbase.namequeues.impl.SlowLogQueueService</value>
+    <value>org.apache.hadoop.hbase.namequeues.impl.SlowLogQueueService,org.apache.hadoop.hbase.namequeues.impl.BalancerDecisionQueueService</value>
```

Comment:
Just a thought:
Can we leave this blank by default? And then do a runtime scan of classes implementing the log queue interface if this param is blank, to include them all by default. Over the long run this would be less prone to error.

Reply:
Ok, sounds good.